server/odc-service/src/main/java/com/oceanbase/odc/service/logger/LoggerService.java

```diff
@@ -122,13 +123,17 @@ public String getLogByTaskFramework(OdcTaskLogLevel level, Long jobId) throws IO
             // process mode when executor is not current host, forward to target
             if (!jobDispatchChecker.isExecutorOnThisMachine(jobEntity)) {
                 ExecutorIdentifier ei = ExecutorIdentifierParser.parser(jobEntity.getExecutorIdentifier());
-                DispatchResponse response = requestDispatcher.forward(ei.getHost(), ei.getPort());
-                return response.getContentByType(new TypeReference<SuccessResponse<String>>() {}).getData();
+                boolean connectable = HttpUtil.isConnectable(ei.getHost(), ei.getPort(), 3);
```

Comment:
magic number used

Reply:
okey, add properties`getCheckOdcServerCanBeConnectedTimes` in `TaskFrameworkProperties`